src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+document.querySelector('.spider').style.cssText = `
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
```

Comment:
It's boring solution, let's use clientWidth and offsetWidth (and the same properties for height)